playbooks/elasticsearch-reindex.yml

```diff
@@ -23,11 +23,18 @@
   hosts: logstash_all
   user: root
   tasks:
-    - name: Service stop
+    - name: Attempt to gracefully stop logstash
       service:
         name: logstash
         state: stopped
         pattern: logstash
+      ignore_errors: true
+    - name: Ensure logstash is stopped
+      shell: "ps -ef | grep -v grep | grep -w logstash | awk '{print $2}'"
+      register: running_processes
+    - name: Force kill logstash if still running
+      shell: "kill -9 {{ item }}"
```

Comment:
Why not just do a pkill -f -9 logstash ?

Reply:
That works too, updated to reduce the code.